PluginDirectories/1/thing.bundle/info.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "Thingiverse",
```

Comment:
for future reference: "name" needs to be the same as the name of the bundle (i.e. `thing`)